### PR TITLE
Refactor ui naming

### LINKED ISSUE #133: Refactor Ui naming
Refactor the Ui naming to suit our new names -> Application, Company, Profile.

### PR DESCRIPTION
**Pull request by seanjyjy**
Refactor the Ui naming to suit our new names -> Application, Company, Profile.

### CODE REVIEW

File: src/main/java/seedu/address/ui/display/TitleDescriptionDisplay.java [hunk 1]
```diff
@@ -22,8 +22,8 @@
     private static final String FXML = "/view/TitleDescriptionDisplay.fxml";
 
     //FXML properties
-    private static final String USER_INFORMATION_WIDTH = "-fx-min-width: 95";
-    private static final String COMPANY_INTERNSHIP_INFORMATION_WIDTH = "-fx-min-width: 75";
+    private static final String PROFILE_INFORMATION_WIDTH = "-fx-min-width: 95";
+    private static final String COMPANY_APPLICATION_INFORMATION_WIDTH = "-fx-min-width: 75";
```
Comment (shawn-nyk) [inline src/main/java/seedu/address/ui/display/TitleDescriptionDisplay.java]: Just to confirm, does company application refer to internship application i.e. our application class in this case?

**Review by shawn-nyk — COMMENTED** [review]
Fantastic work! Just one small question and we're good to merge (after the merge conflict is resolved)!

File: src/main/java/seedu/address/ui/display/TitleDescriptionDisplay.java (hunk 1 above)
Comment (seanjyjy) [reply src/main/java/seedu/address/ui/display/TitleDescriptionDisplay.java]: Thanks for the clarification Shawn, what I meant as this naming is, Company & Application to have the same Information Width. I will do the relevant changes to make it clearer.

Comment (shawn-nyk) [reply src/main/java/seedu/address/ui/display/TitleDescriptionDisplay.java]: Ah I see! Thanks for the clarification Sean!

**Review by shawn-nyk — APPROVED** [review]
LGTM!

**Pull request merged**